phpBB/language/en/composer.json

```diff
@@ -22,6 +22,10 @@
 		"language-iso": "en",
 		"english-name": "British English",
 		"local-name": "British English",
-		"phpbb-version": "4.0.0-a1-dev"
+		"phpbb-version": "4.0.0-a1-dev",
+		"direction": "ltr",
+		"user-lang": "en-gb",
+		"plural-rule": "1",
```

Comment:
It makes sense probably to unquote `"1"` to make it an integer rather than a string.